releases/release-1.30/release-notes/maps/pr-121954-map.yaml

```
@@ -1,8 +1,5 @@
 pr: 121954
 releasenote:
-  text: |-
-    Updated some interfaces' signature in scheduler:
-
-    1. PluginsRunner: used NodeInfo in `RunPreScorePlugins` and `RunScorePlugins`.
-    2. PreScorePlugin: used NodeInfo in `PreScore`.
-    3. Extender: used NodeInfo in `Filter` and `Prioritize`.
+  text: 'Updated some interfaces'' signature in scheduler: - PluginsRunner: used NodeInfo
```

```suggestion
  text: Some interfaces' signatures in the scheduler were updated: PluginsRunner: used NodeInfo in RunPreScorePlugins and RunScorePlugins. PreScorePlugin: used NodeInfo in PreScore. Extender: used NodeInfo in Filter and Prioritize.
```

@rashansmith PTAL at this one. Should we instead use `\n`

---

We can use `|-` here. This helps to maintain the format and avoid breaking the YAML file.